config/management/src/smoke_test.rs

```diff
@@ -139,11 +127,11 @@ fn smoke_test() {
             config.base.waypoint = WaypointConfig::FromStorage(backend);
         }
         config.execution.genesis = Some(genesis.clone());
+        config.execution.genesis_file_location = PathBuf::from("");
```

Comment:
Should we make a file for this to load later too?

Reply:
well I guess the issue is the templates contain some examples, so I'm half considering making actual defaults that contain no examples... or instead to make these functions return really default configs...